docs/appkit/overview.mdx

```diff
@@ -104,13 +112,50 @@ With a layered feature stack spanning onboarding to transactions, messaging and
 
 ![AppKit banner](../../static/img/appkit.png)
 
-Some of the key features of the AppKit SDK include:
+#### Framework Agnostic
+Integrate effortlessly with popular frameworks like React, Vue, Svelte, vanilla JavaScript, and more.
+
+#### EIP-6963 Support
+Seamlessly connect browser extension wallets for an enhanced user experience. Learn more about **EIP-6963 [here](https://eips.ethereum.org/EIPS/eip-6963).**
+
+#### Email Wallets
+Enable users to connect to dapps using just their email address, ensuring a smooth onboarding process.
+
+#### On-Ramp
+Allow users to securely purchase crypto with fiat, directly within your app, supporting over 100 cryptocurrencies.
+
+#### Customizable UI
+Choose between light and dark modes, accent colors, or fully customize the interface with your own branding.
+
+#### Multi-Chain Ready
+Easily configure support for EVM and non-EVM chains, giving users access to multiple blockchain networks.
+
+#### Hundreds of Wallets
+Connect with popular wallets like MetaMask, Rainbow, Coinbase, and many more to maximize accessibility.
+
+#### Transaction History
```

Comment:
you have two times 
#### Transaction History

Reply:
Thanks for catching this. Fixed